src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+	Component,
+	HostBinding,
+	Input,
+	OnInit,
+	TemplateRef,
+	ViewChild
+} from "@angular/core";
+
+@Component({
+	selector: "ibm-breadcrumb-item",
+	template: `
+	<a class="bx--link"
+		href="{{href}}"
+		*ngIf="href; else content">
+		<ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+	</a>
+	<ng-template #content>
+		<ng-content></ng-content>
+	</ng-template>`
+})
+export class BreadcrumbItemComponent implements OnInit {
+	@Input() href: string;
+
+	@Input() contentTemplate: TemplateRef<any>;
+
+	@HostBinding("class") itemClass = "bx--breadcrumb-item";
+
+	@ViewChild("content") template: TemplateRef<any>;
+
+	ngOnInit() {
+		if (!this.contentTemplate) {
+			this.contentTemplate = this.template;
+		}
+	}
+
+	showNothing() {
```

Comment:
I _think_ I get what this is doing, but a quick doc string would help clarify this